pandas/core/window/expanding.py

```diff
@@ -245,12 +250,23 @@ def quantile(self, quantile, interpolation="linear", **kwargs):
     @Substitution(name="expanding", func_name="cov")
     @Appender(_doc_template)
     @Appender(_shared_docs["cov"])
-    def cov(self, other=None, pairwise=None, ddof=1, **kwargs):
+    def cov(
+        self,
+        other: Optional[Union[np.ndarray, FrameOrSeries]] = None,
```

Comment:
do we actually allow other to be a non-frame/series?

Reply:
Yeah we have tests where `other` is an ndarray